### PR TITLE
JOSS paper

### PR DESCRIPTION
**Pull request by arcuri82**
Hi,
 I prepared a paper for [JOSS](https://joss.theoj.org/). The PDF is automatically built online. You can check it from [here](https://whedon.theoj.org/).

Let me know if you have any comment on the paper (e.g., typos).

I will also need your ORCIDs

I am going to make a `1.0.0` release soon (sometime this week). After that, I am planning to submit this paper to JOSS.

### CODE REVIEW

File: docs/joss-paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ - name: FCEyN-UBA, and ICC, CONICET-UBA, Depto. de Computaci\'on, Buenos Aires, Argentina
+   index: 2
+date:  January 2019
+bibliography: paper.bib
+---
+
+
+# Summary
+
+Testing web/enterprise applications is complex and expensive when done manually.
```
Comment (ZhangMan1126) [inline docs/joss-paper/paper.md]: is -> are

File: docs/joss-paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Therefore, in *Software Engineering* (SE) research, lot of effort has been spent in trying
```
Comment (ZhangMan1126) [inline docs/joss-paper/paper.md]: lot -> a lot

File: docs/joss-paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+``EvoMaster`` [@arcuri2018evomaster]  is a SBST tool 
+that automatically *generates* system-level test cases.
+Internally, it uses an *Evolutionary Algorithm* 
+and *Dynamic Program Analysis*  to be able to generate effective test cases.
+The approach is to *evolve* test cases from an initial population of 
+random ones, using code coverage and fault detection as fitness function.
+
+When addressing the testing of real-world web/enterprise applications, there are many challenges. 
+To face and overcome those challenges, EvoMaster has been used to experiment several novel techniques.
```
Comment (ZhangMan1126) [inline docs/joss-paper/paper.md]: experiment -> experiment with?

File: docs/joss-paper/paper.md (hunk 2 above)
Comment (arcuri82) [reply docs/joss-paper/paper.md]: fixed

File: docs/joss-paper/paper.md (hunk 3 above)
Comment (arcuri82) [reply docs/joss-paper/paper.md]: fixed

File: docs/joss-paper/paper.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+To face and overcome those challenges, EvoMaster has been used to experiment with several novel techniques.
+This led to several publications:
+novel search algorithms such as *MIO* [@mio2017][@arcuri2018test],
+addressing the white-box testing of RESTful APIs [@arcuri2017restful][@arcuri2019restful],
+resource-dependency handling [@zhang2019resource], accesses to SQL databases [@arcuri2019sql],
+and novel *testability transformations* [@arcuri2020testability].
+
+
+``EvoMaster`` is aimed both at practitioners that want to automatically test their software, 
+and also at researchers that need generated test cases for the SE problems that they are investigating.
```
Comment (bogdanmarculescu) [inline docs/joss-paper/paper.md]: I don't think the "also" here is needed. The sentence already sets up "both at practitioners [...] and at researchers"

File: docs/joss-paper/paper.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+---
+
+
+# Summary
+
+Testing web/enterprise applications is complex and expensive when done manually.
+Therefore, in *Software Engineering* (SE) research, a lot of effort has been spent in trying 
+to design and implement novel techniques aimed at automating several different tasks in SE.
+*Search-Based Software Testing* (SBST) casts the problem of software testing as an optimization problem,
+aimed at for example maximizing code coverage and fault detection.
```
Comment (bogdanmarculescu) [inline docs/joss-paper/paper.md]: maybe wrap the "for example" between commas "aimed at, for example, maximizing"

**Comment by jgaleotti** [comment]
My ORCID is https://orcid.org/0000-0002-0747-8205
I have no other comments

File: docs/joss-paper/paper.md (hunk 4 above)
Comment (arcuri82) [reply docs/joss-paper/paper.md]: fixed

File: docs/joss-paper/paper.md (hunk 5 above)
Comment (arcuri82) [reply docs/joss-paper/paper.md]: fixed

**Comment by arcuri82** [comment]
thx!
Only @bogdanmarculescu  's ORCID is missing, and then we are ready to go ;)

**Pull request merged**